app/controllers/authors_controller.rb

```diff
@@ -0,0 +1,6 @@
+class AuthorsController < ApplicationController
+  def index
+    # TODO should eager load 5 books only
```

Comment:
Annotation keywords like `TODO` should be all upper case, followed by a colon, and a space, then a note describing the problem.